lib/OpenQA/WebAPI/Plugin/Helpers.pm

```diff
@@ -77,6 +77,12 @@ sub register ($self, $app, $config) {
             return $c->include_branding('external_reporting', %args);
         });
 
+    $app->helper(
+        human_readable_size => sub {
+            my ($c, $size) = @_;
+            return human_readable_size($size);
+        });
```

Comment:
Maybe tidy lets you write a one-liner:
```suggestion
    $app->helper(human_readable_size => sub ($c, $size) { human_readable_size $size });
```

Reply:
It looks it allows it, but there are other functions with just single line which aren't one-liners, thus is it wise to change code style now?

Reply:
I would say yes because I'm not a fan of mass changes.

Reply:
I'd prefer to keep the old style (not one liner).